src/libraries/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -345,43 +345,81 @@ public static bool IsDefined(Type enumType, object value)
         public static object Parse(Type enumType, string value) =>
             Parse(enumType, value, ignoreCase: false);
 
+        public static object Parse(Type enumType, ReadOnlySpan<char> value) =>
```

Comment:
For the new public APIs, please add XML comments.  You can copy/paste from the docs site to seed it.